docs/commandline/pouch_upgrade.md

```diff
@@ -4,88 +4,27 @@ Upgrade a container with new image and args
 
 ### Synopsis
 
-Upgrade a container with new image and args
+upgrade is a feature to replace a container's image.You can specify the new Entrypoint and Cmd for the new container. When you want to updatea container's image, but inherit the network and volumes of the old container, then you shouldthink about the upgrade feature.
```

Comment:
Yes, you are right. Since this is an auto-generated pull request. Could you please help to fix the source doc of https://github.com/alibaba/pouch/blob/master/cli/upgrade.go#L15 to add a white space? 
```
var upgradeDescription = "upgrade is a feature to replace a container's image." +
	"You can specify the new Entrypoint and Cmd for the new container. When you want to update" +
	"a container's image, but inherit the network and volumes of the old container, then you should" +
	"think about the upgrade feature."
```